pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -185,12 +198,14 @@ let
       '';
 
       buildFlags = [
+        "DTC_FLAGS=-@"
```

Comment:
And I moved it out of the `buildDTBs` conditional because I think it's fairly safe to assume that a variable called `DTC_FLAGS` is only going to be used by the devicetree compiler, and if we're not building device tree binaries, the device tree compiler won't be run anyway.